### PR TITLE
Allow initialzer for constant filer like: "let length = 100"

### PR DESCRIPTION
**Pull request by ekazaev**

### CODE REVIEW

File: Tests/FileTests/FieldSpecTests.swift [hunk 1]
```diff
@@ -44,6 +44,26 @@ var hashValue: Int {
         XCTAssertEqual(result, field.toString())
     }
 
+    func testConstantProperty() {
+        /*
+         var hashValue: Int {
```
Comment (NKorchagin) [inline Tests/FileTests/FieldSpecTests.swift]: I think this comment for expected result is out of date from prev method, should be
`let urlString: String = https://www.gilt.com`

**Comment by NKorchagin** [comment]
LGTM

File: Tests/FileTests/FieldSpecTests.swift (hunk 1 above)
Comment (ekazaev) [reply Tests/FileTests/FieldSpecTests.swift]: Shite :)

**Comment by EdSancha** [comment]
👍

**Pull request merged**